base/bin/build-init

```diff
@@ -52,8 +64,13 @@ fi
 
 # Pass build secrets to sandbox
 # A "secret" is any environment variable that starts with "SECRET_"
-secrets="$(env | grep '^SECRET_')" || true
+
+secrets="$(compgen -A variable | grep '^SECRET_')" || true
```

Comment:
Explain what this is doing in a commend. Remove the old commented out line.

Reply:
Added comments surrounding what information is being extracted and how it's being used.